src/Microsoft.Data.Sqlite/SqliteConnection.cs

```diff
@@ -16,6 +17,7 @@ public class SqliteConnection : DbConnection
         private SqliteConnectionStringBuilder _connectionStringBuilder;
         private ConnectionState _state;
         private Sqlite3Handle _db;
+        private bool _sharedCacheMode;
```

Comment:
Technically, this is already cached by `_connectionStringBuilder.CacheMode`, but there's no harm in caching it again.